code/modules/recycling/disposal.dm

```diff
@@ -365,7 +366,7 @@
 	var/atom/L = loc						// recharging from loc turf
 
 	var/datum/gas_mixture/env = L.return_air()
-	var/pressure_delta = (SEND_PRESSURE*1.01) - air_contents.return_pressure()
+	var/pressure_delta = (SEND_PRESSURE*1.05) - air_contents.return_pressure()
```

Comment:
This number is pretty magical

Reply:
Putting the expression back to ONE_AMOSPHERE*2.1 like it was before. Though I don't know what difference it makes.
https://github.com/vgstation-coders/vgstation13/commit/d521357c068fc3cc86a6dcdf339e3fd24f95e8d2#diff-c113cc8b190f978c4e67605c46061b1bL310

Reply:
I have absolutely no idea